### ISSUE TITLE
Use contracts in params

### ISSUE THREAD

**Issue by plozmun** [spatie/schema-org#119]
PHPStan sometimes gives errors like:
```
  31     Parameter #1 $provider of method Spatie\SchemaOrg\Course::provider()     
         expects                                                                  
         array<Spatie\SchemaOrg\Organization|Spatie\SchemaOrg\Person>|Spatie\Sch  
         emaOrg\Organization|Spatie\SchemaOrg\Person,                             
         Spatie\SchemaOrg\EducationalOrganization given.  
```
In this case EducationalOrganization is correct and implements OrganizationContract 
It could be posible to change doc params to use Contracts instead?

**Comment by Gummibeer** [comment]
Good catch - have simply forgot to adjust the doc-tags. Do you want to open a PR that adjusts the generator?

**Comment by plozmun** [comment]
Oh! I was trying but better yours :)

**Comment by Gummibeer** [comment]
Oh, sorry - just had time and you haven't answered.^^
Could you check if the result is what you've expected?

**Comment by plozmun** [comment]
Yeah! It`s ok!

**Comment by plozmun** [comment]
Solved in #120

**Comment by Gummibeer** [comment]
Thanks for your report. It's released now: https://github.com/spatie/schema-org/releases/tag/2.11.0 :)
